.NET/Microsoft.Recognizers.Definitions.Common/English/NumbersDefinitions.cs

```diff
@@ -73,7 +73,7 @@ public static class NumbersDefinitions
 		public static readonly string NumberWithPrefixPercentage = $@"(per cent of|percent of|percents of)(\s*)({BaseNumbers.NumberReplaceToken})";
 		public static readonly string NumberWithPrepositionPercentage = $@"({BaseNumbers.NumberReplaceToken})\s*(in|out\s+of)\s*({BaseNumbers.NumberReplaceToken})";
 		public const string TillRegex = @"(to|through|--|-|—|——|~|–)";
-		public const string MoreRegex = @"((bigger|greater|more|higher|larger)(\s+than)?|above|over|(?<!<|=)>)";
+		public const string MoreRegex = @"((bigger|greater|more|higher|larger|exceed|surpass)(\s+than)?|above|over|(?<!<|=)>)";

```

Comment:
"exceed|surpass" should be with "above|over" in this case. They can't happen before "than"